monster_waves/RandomGenerator.cpp

```diff
@@ -0,0 +1,11 @@
+#include "RandomGenerator.hpp"
+
+RandomNumber::RandomNumber()
+{
+   srand(time(NULL));
+}
+
+int RandomNumber::getNumber(const int min, const int max)
```

Comment:
@JakubMastalski Drobna uwaga, ale zwracaj uwagę na formatowanie. Jeżeli tutaj nie dodajesz przerw między nawiasem, a argumentami to rób tak wszędzie. Spojrzałem na projekt i wszędzie jest inaczej. To nie wygląda elegancko.